plugins/hls-alternate-number-format-plugin/src/Ide/Plugin/AlternateNumberFormat.hs

```diff
@@ -35,7 +35,7 @@ import           Ide.Types
 import qualified Language.LSP.Protocol.Lens       as L
 import           Language.LSP.Protocol.Message
 import           Language.LSP.Protocol.Types
-
+import           Prelude                          hiding (log)
```

Comment:
Personally, I'd prefer to rename the log messages instead of adding an import statement. Anyone else agrees?

Reply:
I saw several plugins name their logging function log and hide Prelude.
I'm not against renaming it to something else if you suggest a reasonable name to rename it to :smile:

Reply:
Yeah, I know a number of plugins are doing that.

I like `msg` when you just pass something through such as `ShakeLog`. But it can also be the more wordy `shakeLog` or even `l`, as the name is not giving you much in this particular situation.

Otherwise, we usually don't have something such as `Log String`, but more concrete types in our logging constructors. These concrete types can have more descriptive names, just like in a regular function.

Reply:
Alright, renamed to `msg`